modules/RTC/TPCUtils.js

```diff
@@ -723,10 +723,14 @@ export class TPCUtils {
 
         return videoCodec === CodecMimeType.VP8 // VP8 always
 
-            // K-SVC mode for VP9 when no scalability mode is set. Though only one outbound-rtp stream is present,
-            // three separate encodings have to be configured.
+            // For FF: scalabilityModeEnabled is not supported and we have to use simulcast.
+            // For other browsers we use K-SVC mode for VP9 when no scalability mode is set. Although
+            // only one outbound-rtp stream is present, three separate encodings have to be configured.
             || (!this.codecSettings[videoCodec].scalabilityModeEnabled && videoCodec === CodecMimeType.VP9)
 
+            // FF uses simulcast with AV1.
```

Comment:
Do we know how that performs?

Reply:
I haven't done comparison on CPU usage, etc with VP8 if this is what you mean. Otherwise during testing I haven't noticed any issues. 

If I have to speculate it should be fine since we had it enabled and on FF we don't have other codec that will use SVC, all of them will be using simulcast....

Do you want me to compare it more formally with VP8 and post the results? How do we normally do this?

Reply:
I'd rather use the setting from `useSimulcast` from codec settings to determine if the encoder is configured to use simulcast. This way it is easy to just flip the flag in config.js to test if the Firefox bug is fixed.
`this.codecSettings[videoCodec].scalabilityModeEnabled && this.codecSettings[videoCodec].useSimulcast`